### PR TITLE
chore(Core/Database): Add info about errors

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:
- Point to wiki... Hopefully less tickets from people that do weird stuff o.o

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Winfidonarleyan — APPROVED** [review]

**Comment by Winfidonarleyan** [comment]
Maybe need add url `https://www.azerothcore.org/wiki/common-errors` ?

**Comment by Kitzunu** [comment]
Nah, it is easily found on the wiki

**Comment by Winfidonarleyan** [comment]
> Nah, it is easily found on the wiki

There are all kinds of people :D

**Comment by Kitzunu** [comment]
Also true... But I dont want to make the message too long either

**Comment by Winfidonarleyan** [comment]
> Also true... But I dont want to make the message too long either

just add new lines

**Comment by Kitzunu** [comment]
I dont remember how Q.Q

**Comment by Winfidonarleyan** [comment]
> I dont remember how Q.Q

`\n`

File: src/server/database/Database/DatabaseWorkerPool.cpp [hunk 1]
```diff
@@ -46,8 +46,8 @@ DatabaseWorkerPool<T>::DatabaseWorkerPool()
       _async_threads(0), _synch_threads(0)
 {
     WPFatal(mysql_thread_safe(), "Used MySQL library isn't thread-safe.");
-    WPFatal(mysql_get_client_version() >= MIN_MYSQL_CLIENT_VERSION, "AzerothCore does not support MySQL versions below 5.7");
-    WPFatal(mysql_get_client_version() == MYSQL_VERSION_ID, "Used MySQL library version (%s id %lu) does not match the version id used to compile AzerothCore (id %u)",
+    WPFatal(mysql_get_client_version() >= MIN_MYSQL_CLIENT_VERSION, "AzerothCore does not support MySQL versions below 5.7. Search the wiki for ACE00043 in Common Errors.");
```
Comment (Winfidonarleyan) [inline src/server/database/Database/DatabaseWorkerPool.cpp:49]:
```suggestion
    WPFatal(mysql_get_client_version() >= MIN_MYSQL_CLIENT_VERSION, "AzerothCore does not support MySQL versions below 5.7. \nSearch the wiki for ACE00043 in Common Errors.");
```

**Comment by Kitzunu** [comment]
Ah yes ofc. I fix this tomorrow

**Review by FrancescoBorzi — APPROVED** [review]

**Review by FrancescoBorzi — APPROVED** [review]

**Pull request merged**